oca_dependencies.txt

```diff
@@ -27,11 +27,11 @@ partner-contact
 queue
 report-print-send
 server-ux
+social https://github.com/CompassionCH/social 11.0-mig-mail_sendgrid
```

Comment:
At line 25, geospatial should use the fork of josuebulle:
`geospatial https://github.com/josueBulle/geospatial 11.0-mig-geoengine_partner`